### PR TITLE
update translation strings

### PR DESCRIPTION
**Pull request by vellip**
not done yet

### CODE REVIEW

File: locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -339,11 +339,11 @@ msgstr "Ihr Name"
 
 #: apps/bplan/models.py:38 apps/users/models.py:27
 msgid "Email address"
-msgstr "E-Mail Adresse"
+msgstr "E-Mail Addresse"
```
Comment (xi) [inline locale/de/LC_MESSAGES/django.po]: This is actually wrong.

File: locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -121,7 +121,8 @@ msgstr "Bitte setzen Sie einen Punkt auf der Karte."
 #: adhocracy4/maps/fields.py:39
 #: adhocracy4/maps/fields.py:39
 msgid "Please draw a Polygon on the map"
-msgstr "Bitte zeichnen Sie einen Bereich auf der Karte ein."
+msgstr "Bitte zeichnen Sie einen Bereich auf der Karte ein oder wählen Sie einen "
+"vordefinierten Bereich aus"
```
Comment (xi) [inline locale/de/LC_MESSAGES/django.po]: We should adapt the source string, too.

File: locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -355,11 +356,11 @@ msgstr "Postleitzahl, Stadt"
 
 #: apps/bplan/phases.py:15
 msgid "Statement phase"
-msgstr "Stellungsnahmephase"
+msgstr "Beteiligungsphase"
 
 #: apps/bplan/phases.py:16
 msgid "Send statement to the office workers per mail."
-msgstr "Stellungnahme per E-Mail an Mitarbeiter senden "
+msgstr "Stellungnahme per E-Mail an Verwaltung senden "
```
Comment (xi) [inline locale/de/LC_MESSAGES/django.po]: Should end with a `.`

File: locale/de/LC_MESSAGES/django.po (hunk 2 above)
Comment (vellip) [inline locale/de/LC_MESSAGES/django.po]: That's in core, and that functionality is not in core.

Comment (xi) [inline locale/de/LC_MESSAGES/django.po]: How unfortunate…

**Comment by xi** [comment]
I found some remaining issue, but i will merge this now so we can fix them seperately:

- For some reason the image help text translation is not displayed
- The polygon input show "Enter valid JSON" as help text
- The german allauth translations use informal "Du" (e.g. the flash message on logout)

**Pull request merged**